.github/governance.yml

```diff
@@ -0,0 +1,41 @@
+version: v1
+
+pull_request:
+  labels:
+    - prefix: kind
+      multiple: false
+      list: ['feature', 'fix', 'chore', 'docs', 'refactor', 'dependencies']
+      needs:
+        comment: |
+          @$AUTHOR: There are no 'kind' label on this PR. You need a 'kind' label to generate the release automatically.
+
+          * `/kind feature`
```

Comment:
I suggest we follow the format of convetional commits [list](https://www.npmjs.com/package/@commitlint/config-conventional). Full spec https://www.conventionalcommits.org/en/v1.0.0/

Reply:
https://github.com/DeFiCh/oss-governance-bot#oss-governance-bot

Reply:
Why I am against it is because it makes PR more complicated for the average users, we want to encourage all community contributors from <1 YOE to 20 YOE engineers.

Even a small semantic or translation correction, the contributor should not have to understand or governance policy. The governance policy should be educational by design.

Reply:
> Although putting our project on GitHub makes it transparent and visible for public contributions, it's far from making it maintainable. For new contributors, creating an issue or pull request and successfully sending it is a mountainous journey in itself. Quality control hierarchy such as 'Triage', 'Conventional Commits', 'CI steps' and 'Code review' often deter contributors from contributing due to the complex nature of each OSS governance policy.

For reference

Reply:
I was just referring to the labels list. Example, ['build', 'ci', 'chore', 'docs', 'feat',  'fix',  'perf',  'refactor',  'revert',  'style',  'test'];. I think these labels defined by conventional commits is widely used by most projects. Anyway, just a suggestion I'm ok with using either.